### PR TITLE
Try to wrap the messages appropriately to look better on screen

### PR DESCRIPTION
**Pull request by rfay**
## The Issue

Too much wrapping on messages sent to the user.

## How This PR Solves The Issue

Try to calm them down.

### CODE REVIEW

File: remote-config.jsonc [hunk 1]
```diff
@@ -16,16 +16,16 @@
 			"interval": 20,
 			"messages": [
 				{
-					"message": "We’re actively looking for financial sponsors! https://github.com/sponsors/ddev"
+					"message": "You can make DDEV last by becoming a financial sponsor!\nhttps://github.com/sponsors/ddev"
```
Comment (gilbertsoft) [inline remote-config.jsonc:19]: Fine with me to break URLs

File: remote-config.jsonc [hunk 2]
```diff
@@ -16,16 +16,16 @@
...
 				},
 				{
-					"message": "A 403 from your project may mean the docroot is not specified or `index.php` is missing."
+					"message": "A 403 from your project may mean that\nthe docroot is not specified or `index.php` is missing."
```
Comment (gilbertsoft) [inline remote-config.jsonc:22]: I don't like to introduce line breaks in the middle ot the text, console should take care about correct wrapping instead imho.

File: remote-config.jsonc [hunk 3]
```diff
@@ -16,16 +16,16 @@
...
 				},
 				{
-					"message": "You can have as many databases as you want! https://ddev.readthedocs.io/en/latest/users/usage/faq/#can-i-use-additional-databases-with-ddev"
+					"message": "You can have as many databases as you want!\nhttps://ddev.readthedocs.io/en/latest/users/usage/faq/#can-i-use-additional-databases-with-ddev"
 				},
 				{
-					"message": "Use `ddev describe` to get a summary of your project’s configuration."
+					"message": "Use `ddev describe` to get a summary\nof your project’s configuration."
```
Comment (gilbertsoft) [inline remote-config.jsonc:28]: Same as above

File: remote-config.jsonc [hunk 4]
```diff
@@ -34,25 +34,25 @@
...
-					"message": "`ddev snapshot` is great to get a quick copy of your database. `ddev help snapshot`"
+					"message": "`ddev snapshot` makes a quick backup of your database.\n`ddev help snapshot`"
 				},
 				{
-					"message": "Try `ddev share` so collaborators anywhere can see your current project. https://ddev.readthedocs.io/en/latest/users/topics/sharing/"
+					"message": "Try `ddev share` so collaborators anywhere can see your current project.\nhttps://ddev.readthedocs.io/en/latest/users/topics/sharing/"
 				},
 				{
-					"message": "`ddev poweroff` is a great way to get back to the beginning and save resources."
+					"message": "`ddev poweroff` is a great way to get back to the beginning\nand save resources."
```
Comment (gilbertsoft) [inline remote-config.jsonc:43]: Same as above

File: remote-config.jsonc [hunk 5]
```diff
@@ -34,25 +34,25 @@
...
-					"message": "Different projects can communicate with each other! https://ddev.readthedocs.io/en/latest/users/usage/faq/#can-different-projects-communicate-with-each-other"
+					"message": "Different projects can communicate with each other!\nhttps://ddev.readthedocs.io/en/latest/users/usage/faq/#can-different-projects-communicate-with-each-other"
 				},
 				{
-					"message": "The FAQ is a useful collection of not-obvious details: https://ddev.readthedocs.io/en/latest/users/usage/faq"
+					"message": "The FAQ is a useful collection of not-obvious details:\nhttps://ddev.readthedocs.io/en/latest/users/usage/faq"
 				},
 				{
-					"message": "Use `ddev composer` instead of `composer` to ensure your project uses the Composer and PHP versions it specifies."
+					"message": "Use `ddev composer` instead of `composer` to ensure your project\nuses the Composer and PHP versions it specifies."
```
Comment (gilbertsoft) [inline remote-config.jsonc:55]: Same as above

**Review by gilbertsoft — CHANGES_REQUESTED** [review]

**Comment by rfay** [comment]
Let's skip this and wrap in the code then, good idea.

**Pull request closed without merging**